File: api/v2010/message/create-test-sms-messages-example-3/create-test-sms-messages-example-3.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$message = $twilio->messages
                  ->create("+14108675310", // to
                           array(
                               "from" => "+15005550006",
                               "body" => "Do. Or do not. There is no try."
                           )
                  );

print($message->sid);

File: api/v2010/message/create-test-sms-messages-example-3/create-test-sms-messages-example-3.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var message = MessageResource.Create(
            from: new Twilio.Types.PhoneNumber("+15005550006"),
            body: "Do. Or do not. There is no try.",
            to: new Twilio.Types.PhoneNumber("+14108675310")
        );

        Console.WriteLine(message.Sid);
    }
}
